mne/viz/topo.py

```diff
@@ -22,13 +22,13 @@
 from ..fixes import normalize_colors
 from ..utils import _clean_names
 
-from .utils import _mutable_defaults, _check_delayed_ssp, COLORS
+from .utils import _mutable_defaults, _check_delayed_ssp  # , COLORS
 from .utils import _draw_proj_checkbox
 
 
-def iter_topography(info, layout=None, on_pick=None, fig=None,
-                    fig_facecolor='k', axis_facecolor='k',
-                    axis_spinecolor='k', layout_scale=None):
+def iter_topography(pos=None, info=None, on_pick=None, fig=None,
```

Comment:
you haven't documented this new parameter (I know documentation is on your todo list :)